paasta_tools/utils.py

```diff
@@ -2319,6 +2321,25 @@ def get_log_reader(self) -> LogReaderConfig:
                 % self.directory
             )
 
+    def get_log_readers(self) -> List[LogReaderConfig]:
+        """Get the log_readers configuration out of global paasta config
+
+        :returns: the log_readers list of dicts.
+        """
+        try:
+            return self.config_dict["log_readers"]
+        except KeyError:
+            raise PaastaNotConfiguredError(
+                "Could not find log_readers in configuration directory: %s"
+                % self.directory
+            )
+
+    def get_log_readers_migration_status(self) -> Optional[Mapping[str, bool]]:
+        """
+        Get the log readers migration configuration out of global paasta config
+        """
+        return self.config_dict.get("log_readers_migration_status")
```

Comment:
this would need a config item, like:
```
"log_readers_migration_status": {
    "cluster-1-dev": true,
    "cluster-2-prod": false
  },
```

Reply:
@nemacysts wdyt about having this while we are gradually migrating to the new pipeline?

Reply:
i think something like `use_vector_logs` as a config name would be easier to reason about in the code/as a human - but in general seems like a good idea :)

Reply:
renamed a bit, to be less yelpy ))